docs/docs/50-user-guide/70-annotations.md

```diff
@@ -0,0 +1,74 @@
+---
+sidebar_label: Annotations
+description: List of Kargo-specific annotations and labels
+---
+
+# Annotations and Labels used by Kargo
+
+This page documents Kargo-specific annotations, labels, and finalizers. Some of these fields can be set by users, while others are system-managed and used internally by the control plane.
+
+## Annotations
+
+| Annotation key | Target resource(s) | Possible values | Description |
+|----------------|--------------------|------------------|-------------|
+| `kargo.akuity.io/refresh` | Any Kargo resource | Arbitrary string (acts as a token) | Triggers reconciliation when the value changes. Useful for forcing a resync. |
+| `kargo.akuity.io/reverify` | `Stage` | Verification ID or JSON object (VerificationRequest) | Triggers re-verification of the currently assigned Freight. |
+| `kargo.akuity.io/abort` | `Stage` | Verification ID | Aborts an in-progress Freight verification. |
+| `kargo.akuity.io/description` | Any | Free-form string | Human-readable description displayed in the UI. |
+| `kargo.akuity.io/color` | `Stage` | Hex color code (e.g. `#ff8800`) | Optional cosmetic color used in UI. |
+| `kargo.akuity.io/create-actor` | All | Actor identity (username/service) | Injected by the control plane to indicate who created the resource. |
```

Comment:
I'm questioning, especially given the admonishment later on this same page _not_ to mess with system-assigned annotations, whether this isn't actually information that we're better off omitting, at least until such time we decide there's greater value in documenting them.

For now, I'd cut this and we can revisit later if need be.